src/engraving/libmscore/lyrics.h

```diff
@@ -92,7 +92,7 @@ class Lyrics final : public TextBase
     QString subtypeName() const override { return QObject::tr("Verse %1").arg(_no + 1); }
     void setNo(int n) { _no = n; }
     int no() const { return _no; }
-    bool isEven() const { return _no % 1; }
+    bool isEven() const { return _no % 2; }
```

Comment:
What is actually meant here is `_no & 1`
See #7628 (for 3.x, got merged) and #7627 (got closed)

Reply:
Got it, although wouldn't `_no & 1` give the same results as `_no % 2`? [Godbolt](https://godbolt.org/z/P9P9381oG) seems to show the same assembly generated for both.

Reply:
`_no & 1`is used in other places, for the very same purpose. So using it here too is a) consistent and b) the save bet